src/3_branch.js

```diff
@@ -285,6 +285,7 @@ Branch.prototype['init'] = wrap(callback_params.CALLBACK_ERR_DATA, function(done
 			data = setBranchValues(data);
 			if (CORDOVA_BUILD || TITANIUM_BUILD) { // jshint undef:false
 				var first = self._storage.getAll();
+				// I feel this check is wrong?
```

Comment:
The first check still doesn't work as expected. I think there's something up with the logic here.